decidim-core/app/models/decidim/participatory_process.rb

```diff
@@ -33,9 +33,14 @@ class ParticipatoryProcess < ApplicationRecord
     validates :slug, presence: true
     validates :slug, uniqueness: { scope: :organization }
 
+    has_many :components, foreign_key: "decidim_participatory_process_id"
+
     mount_uploader :hero_image, Decidim::HeroImageUploader
     mount_uploader :banner_image, Decidim::BannerImageUploader
 
+    has_many :components, through: :features
```

Comment:
Can we group all the associations in the same block? Also, I'd flip the order between components & features.